include/deal.II/base/config.h.in

```diff
@@ -108,9 +108,6 @@
  */
 
 #cmakedefine DEAL_II_HAVE_CXX11_IS_TRIVIALLY_COPYABLE
-#cmakedefine DEAL_II_HAVE_ISNAN
-#cmakedefine DEAL_II_HAVE_STD_ISNAN
-#cmakedefine DEAL_II_HAVE_UNDERSCORE_ISNAN
```

Comment:
@bangerth Yes, cmake expands the `#cmakedefine` statement with the help of cmake variables (in this case these three variables were set in one of our `check_*.cmake` files.)